ember-browser-services/src/test-support/window-mock-augments.ts

```diff
@@ -1,56 +0,0 @@
-import window from 'ember-window-mock';
-
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-expect-error
-import locationFactory from 'ember-window-mock/test-support/-private/mock/location';
-
-const AUGMENTS: Array<string | symbol> = ['origin'];
-
-function createLocation(target?: Window) {
-  let initialHref = target?.location?.href ?? window.location.href;
-  let mockLocation = locationFactory(initialHref);
-  let values: any = {};
-
-  mockLocation.isPatchedLocation = true;
-
-  return new Proxy(mockLocation, {
-    get(target, key, receiver) {
-      if (AUGMENTS.includes(key)) {
-        return values[key] ?? Reflect.get(target, key, receiver);
-      }
-
-      return Reflect.get(target, key, receiver);
-    },
-
-    set(target, key, value, receiver) {
-      if (AUGMENTS.includes(key)) {
-        return (values[key] = value);
-      }
-
-      return Reflect.set(target, key, value, receiver);
-    },
-  });
-}
-
-export function patchWindow(target: any) {
-  let location = createLocation(target);
-
-  let self: any = new Proxy(target, {
-    get(target, key, receiver) {
-      if (key === 'location') return location;
-      if (key === 'parent') return self;
-      if (key === 'top') return self;
-
-      return Reflect.get(target, key, receiver);
-    },
-    set(target, key, value, receiver) {
-      if (key === 'location') {
-        throw new Error(`location cannot be set on window`);
```

Comment:
This is not correct actually. See https://developer.mozilla.org/en-US/docs/Web/API/Window/location

> Though Window.location is a read-only Location object, you can also assign a string to it. This means that you can work with location as if it were a string in most cases: location = 'http://www.example.com' is a synonym of location.href = 'http://www.example.com'.

`ember-window-mock` itself always allowed this behaviour, see https://github.com/simonihmig/ember-window-mock/blob/master/test-app/tests/unit/run-window-tests.ts#L123-L129